src/tests/test_resultset.py

```diff
@@ -78,6 +78,20 @@ def test_resultset_repr_html(result_set):
         "<tr>\n            <td>0</td>\n        </tr>\n        <tr>\n            "
         "<td>1</td>\n        </tr>\n        <tr>\n            <td>2</td>\n        "
         "</tr>\n    </tbody>\n</table>"
+        "<div><code>ResultSet</code> : "
+        "to convert to pandas, call <code>.DataFrame()</code> "
+        "or to polars, call <code>.PolarsDataFrame()</code></div>"
+    )
+
+
+def test_invalid_operation_error(result_set):
```

Comment:
Can we use parameters here and test different invalid functions (e.g. `tail`, `head(-1)`, `value_counts`, `not_df_function`)?